frontend/src/v5/store/tickets/card/ticketsCard.selectors.ts

```diff
@@ -23,8 +23,11 @@ import { ITicketsCardState } from './ticketsCard.redux';
 import { DEFAULT_PIN, getPinColorHex, formatPin, getTicketPins } from '@/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.helpers';
```

Comment:
Standard `oneOf` property incorrectly becomes a `jobsAndUsers` property when I try to edit it

https://github.com/user-attachments/assets/8c22151a-acb1-482a-82a3-898e35e50e65